crankshaft-monitor/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "crankshaft-monitor"
+version = "0.1.0"
+license.workspace = true
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+tokio.workspace = true
+prost = "0.13.5"
+tonic = "0.13.1"
+futures-util = "0.3.31"
+async-stream = "0.3.6"
+futures-core = "0.3.31"
```

Comment:
We can add these to the root `Cargo.toml` and reference as workspace dependencies here.